.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  merge_group:
+
+jobs:
+  backward_compat:
+    name: "Backward Compatibility"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: run tests/sanity/backward_compatible.py
```

Comment:
The reason why I went this way is because I’m not sure how github deals with changing directories; but I just found out how to tell it to run the whole action in another directory, so this is now handled :) I still left it in the messy "sanity checks" job